geode-core/src/test/java/org/apache/geode/internal/JarDeployerIntegrationTest.java

```diff
@@ -144,40 +131,99 @@ public void run() {
       Thread.sleep(500);
       alternateDir.mkdir();
       thread.join();
-    } catch (Exception e) {
-      fail(e);
+    } catch (InterruptedException iex) {
+      fail("Interrupted while waiting.");
```

Comment:
I made sure to avoid this antipattern in the tests I added, but I had left some old tests around unchanged.  I'll go back and clean them up today.